### PR TITLE
Replace macros Bug and WebkitBug by regular MD links

### PR DESCRIPTION
**Pull request by teoli2003**
Content counterpart of mdn/yari#8298

### CODE REVIEW

File: files/en-us/glossary/quality_values/index.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Some syntax, like the one of {{HTTPHeader("Accept")}}, allow additional specifie
 
 ### Firefox
 
-Starting with Firefox 18, the quality factor values are clamped to 2 decimal places. They used to be clamped to only 1 decimal place in earlier versions ({{bug(672448)}}).
+Starting with Firefox 18, the quality factor values are clamped to 2 decimal places. They used to be clamped to only 1 decimal place in earlier versions ([Firefox bug 672448](https://bugzilla.mozilla.org/show_bug.cgi?id=672448)).
```
Comment (queengooborg) [inline files/en-us/glossary/quality_values/index.md:45]: Could we use the shortlinks instead, i.e. `https://bugzil.la`?

```suggestion
Starting with Firefox 18, the quality factor values are clamped to 2 decimal places. They used to be clamped to only 1 decimal place in earlier versions ([Firefox bug 672448](https://bugzil.la/672448)).
```

Comment (teoli2003) [reply files/en-us/glossary/quality_values/index.md:45]: Why? This adds a redirection and an extra request:
Capture d’écran 2023-02-24 à 11 25 41

Comment (queengooborg) [reply files/en-us/glossary/quality_values/index.md:45]: bugzil.la, webkit.org/b and crbug.com are shorter and thus, much easier to read (which helps contributors).  I don't know about MDN as a whole, but BCD has a policy in place to use shortlinks for bug links.  (Maybe someone can confirm if there is an MDN-wide policy for these cases.)

Comment (teoli2003) [reply files/en-us/glossary/quality_values/index.md:45]: There isn't but I have no problem using them. No reason to do differently.

Let me fix this

**Comment by teoli2003** [comment]
Fixed. Note that as we don't have linting on this, this will diverge; I don't think it is a problem, though.

File: files/en-us/mdn/writing_guidelines/page_structures/macros/commonly_used_macros/index.md [hunk 2]
```diff
@@ -165,10 +165,10 @@ Most macros will also take a second argument allowing you to change the display
 
 - Bugs
```
Comment (queengooborg) [inline files/en-us/mdn/writing_guidelines/page_structures/macros/commonly_used_macros/index.md:166]: We should actually remove this entire section; it's talking strictly about the bug macros which will be removed soon!

**Review by queengooborg — APPROVED** [review]

**Pull request merged**